plugins/extra/bulkcnam.go

```diff
@@ -0,0 +1,84 @@
+package extra
+
+import (
+	"io"
+	"bufio"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"unicode"
+
+	"github.com/belak/go-seabird"
+	"github.com/go-irc/irc"
+)
+
+func init() {
+	seabird.RegisterPlugin("bulkcnam", newBulkCNAMPlugin)
+}
+
+type bulkCNAMPlugin struct {
+	Key string
+}
+
+func newBulkCNAMPlugin(b *seabird.Bot, cm *seabird.CommandMux) error {
+	p := &bulkCNAMPlugin{}
+
+	err := b.Config("bulkcnam", p)
+	if err != nil {
+		return err
+	}
+
+	cm.Event("cnam", p.bulkCNAMCallback, &seabird.HelpInfo{
+		Usage:       "<phone #>",
+		Description: "Returns the CNAM of a phone number",
+	})
+
+	return nil
+}
+
+func (p *bulkCNAMPlugin) bulkCNAMCallback(b *seabird.Bot, m *irc.Message) {
```

Comment:
This function could probably be merged into the BulkCNAM function below, as this one is fairly simple.